draft-ietf-quic-recovery.md

```diff
@@ -867,7 +867,9 @@ between the oldest and newest lost packets: (3 - 0) = 3.  The duration for
 persistent congestion is equal to: (1 * kPersistentCongestionThreshold) = 3.
 Because the threshold was reached and because none of the packets between the
 oldest and the newest packets are acknowledged, the network is considered to
-have experienced persistent congestion.
+have experienced persistent congestion. Note that this example shows the
+occurrence of PTOs, but that is not required for persistent congestion to be
+established.
```

Comment:
Won't this conflict with the other change?

Reply:
Yup, I'll prefer that text when merging.